get_embeddings.py

```diff
@@ -0,0 +1,101 @@
+import clip
```

Comment:
Might be better to move this entire script to within habitat_uncertainty utils folder

Reply:
This script only creates embedding and saves to a pickle file. It is not used throughout the run. But will definitely move it to habitat_uncertainty folder.